cmd/kubelet/app/server.go

```diff
@@ -253,7 +253,11 @@ func (s *KubeletServer) Run(_ []string) error {
 		glog.Warning(err)
 	}
 
-	client, err := s.createAPIServerClient()
+	var apiclient *client.Client
+	clientConfig, err := s.CreateAPIServerClientConfig()
+	if err != nil {
```

Comment:
ugh! you're right ... I accidentally blew away my first change for this, and when I re-wrote it I reversed the logic. will remedy ASAP